### PR TITLE
Hotfix Python Update CI

### PR DESCRIPTION
**Pull request by scbedd**
Ugh.

@praveenkuttappan

### CODE REVIEW

File: eng/common/scripts/update-docs-ci.ps1 [hunk 1]
```diff
@@ -59,7 +59,7 @@ function UpdateParamsJsonPython($pkgs, $ciRepo, $locationInDocRepo){
   foreach ($releasingPkg in $pkgs) {
     if ($visibleInCI.ContainsKey($releasingPkg.PackageId)) {
       $packagesIndex = $visibleInCI[$releasingPkg.PackageId]
-      $existingPackageDef = $targetData[$packagesIndex]
```
Comment (scbedd) [inline eng/common/scripts/update-docs-ci.ps1:62]: Can't reference an object that doesn't exist :/

File: eng/common/scripts/update-docs-ci.ps1 [hunk 2]
```diff
@@ -59,7 +59,7 @@ function UpdateParamsJsonPython($pkgs, $ciRepo, $locationInDocRepo){
...
+      $existingPackageDef = $allJson.packages[$packagesIndex]
```
Comment (weshaggard) [inline eng/common/scripts/update-docs-ci.ps1:62]: Please ensure this change makes it into the tools repo before the next sync.

Comment (scbedd) [reply eng/common/scripts/update-docs-ci.ps1:62]: That's a given 👍

**Pull request merged**